composer.json

```
@@ -27,7 +27,7 @@
         "doctrine/persistence": "^2.1 || ^3.0",
         "sonata-project/admin-bundle": "^4.15",
         "sonata-project/block-bundle": "^4.16",
-        "sonata-project/doctrine-extensions": "^1.8 || ^2.0",
+        "sonata-project/doctrine-extensions": "1.x-dev as 1.18",
```

```suggestion
        "sonata-project/doctrine-extensions": "^1.18 || ^2.1",
```

It can be applied after generate new releases in that bundle!